src/dmd/root/aav.d

```diff
@@ -121,6 +127,63 @@ extern (C++) Value dmd_aaGetRvalue(AA* aa, Key key)
     return null; // not found
 }
 
+debug
+{
+    /**
+    Returns a range of key/values for `aa`.
+    Returns: a range of key/values for `aa`.
```

Comment:
duplicate?

Reply:
Actually it was intentional, I was trying to make a valid ddoc comment.  A description and a return value...maybe one of them is not necessary?